### PR TITLE
Crowdmap

### PR DESCRIPTION
**Pull request by QwindorRousseau**
Added in the new links which i tested on my profile to ensure they were working for the hopefully new github hosting of the crowdmap

### CODE REVIEW

**Comment by keneanung** [comment]
The changes look good, however, there seem to be whitespace changes in every line...

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -8554,7 +8554,7 @@ end
 function mmp.retrievecrowdchangelog()
   mmp.crowdchangelogfile = downloadfolder..&quot;crowdchangelogfile&quot;
 
-  downloadFile(mmp.crowdchangelogfile, &quot;http://vadisystems.com/achaea-mudlet-map/changelog&quot;)
+  downloadFile(mmp.crowdchangelogfile, &quot;http://ire-mudlet-mapping.github.io/AchaeaCrowdmap/Map/changelog.tx&quot;)
```
Comment (keneanung) [inline mudlet-mapper.xml]: shouldn't this be `changelog.txt` (note the last `t`)?

**Comment by QwindorRousseau** [comment]
Ok hopefully all done finally

**Comment by keneanung** [comment]
:+1: Looks good for me

**Comment by vadi2** [comment]
I can't read the changes on my phone with the entire file updates guys

**Comment by keneanung** [comment]
Qwindor fixed it. Some trailing whitespace was removed, but the rest is fine.

**Comment by vadi2** [comment]
https://github.com/vadi2/ire-mapping-script/pull/8.patch still shows the
entire file for me

**Comment by keneanung** [comment]
Because patch shows the whole history of changes (all commits within this PR). Use the diff for the resulting change: https://patch-diff.githubusercontent.com/raw/vadi2/ire-mapping-script/pull/8.diff

**Comment by vadi2** [comment]
Thanks!

+1

**Pull request merged**